### PR TITLE
docs: update changelog and docs

### PR DESCRIPTION
**Pull request by BryanttV**
## Description

This PR updates the changelog and compatibility notes.

- As the bump version workflow has been disabled, the last changes corresponding to the integration test feature must be added to the changelog.
- The compatibility notes must be updated due to an update of the users backend. `v11.7.0` must be used for Palm and Quince.

## Related Issues
- [Jira (DS-1011)](https://edunext.atlassian.net/browse/DS-1011)

### CODE REVIEW

**Review by MaferMazu — APPROVED** [review]
Looks good to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v11.7.0](https://github.com/eduNEXT/eox-tenant/compare/v11.6.0...v11.7.0) - (2024-06-19)
 
+### Added
+
+- **Integration Tests**: A new GitHub workflow was added to run the integration
```
Comment (magajh) [inline CHANGELOG.md:12]:
```suggestion
- **Integration Tests**:  A new GitHub workflow has been added to run integration tests. Integration tests have been included to validate backend imports and to ensure the /eox-info endpoint works correctly.
```

**Review by magajh — APPROVED** [review]
LGTM, just a minor rewording suggestion

**Comment by BryanttV** [comment]
Hi @MaferMazu @magajh, I updated the changelog with the latest suggestions, could you help me with the review again? thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v11.7.0](https://github.com/eduNEXT/eox-tenant/compare/v11.6.0...v11.7.0) - (2024-06-19)
 
+### Added
+
+- **Integration Tests**: A new GitHub workflow has been added to run
+  integration tests. Integration test has been included to validate backend
```
Comment (magajh) [inline CHANGELOG.md:13]: @BryanttV, I’m not sure if just saying "Integration test has been included ..." makes a lot of sense. How about adding "An" to the beginning? For example: "An integration test has been included ..."

Alternatively, we could say:
"A new GitHub workflow has been added to run integration tests. These tests validate backend imports and ensure the `/eox-info` endpoint functions correctly."

Comment (BryanttV) [reply CHANGELOG.md]: Thank you, I took your suggestion

**Review by magajh — APPROVED** [review]

**Review by MaferMazu — APPROVED** [review]

**Pull request merged**